thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/rootcause/impl/FrameworkResultUtils.java

```diff
@@ -0,0 +1,26 @@
+package com.linkedin.thirdeye.rootcause.impl;
+
+import com.linkedin.thirdeye.rootcause.Entity;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+
+public class FrameworkResultUtils {
+  public static Map<EntityUtils.EntityType, Collection<Entity>> topKPerType(Collection<Entity> entities, int k) {
+    Map<EntityUtils.EntityType, Collection<Entity>> map = new HashMap<>();
+    for(Entity e : entities) {
```

Comment:
+1 on variable names. Please evaluate the need for a util class to achieve this functionality.